File: apkEditor/src/main/java/com/mcal/apkeditor/ui/fulleditor/StringsFragment.java
package com.mcal.apkeditor.ui.fulleditor;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.mcal.apkeditor.databinding.FragmentStringsBinding;

public class StringsFragment extends Fragment {
    private FragmentStringsBinding binding;

    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {

        binding = FragmentStringsBinding.inflate(inflater, container, false);
        View root = binding.getRoot();

        return root;
    }
}

File: apkEditor/src/main/java/com/mcal/apkeditor/ui/fulleditor/strings/StringsFragment.kt
package com.mcal.apkeditor.ui.fulleditor.strings

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.fragment.app.activityViewModels
import com.mcal.apkeditor.databinding.FragmentStringsBinding
import com.mcal.apkeditor.ui.fulleditor.FullEditorViewModel

class StringsFragment : Fragment() {
    private lateinit var binding: FragmentStringsBinding
    private val model: FullEditorViewModel by activityViewModels()

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        binding = FragmentStringsBinding.inflate(inflater, container, false)

        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }
}